3rdparty/CMakeLists.txt

```diff
@@ -117,31 +117,44 @@ if (IRR_BUILD_CEGUI)
 
 	if (MSVC)
 		set(CEGUI_LIBS
-			"${CEGUI_INSTALL_PATH}/lib/CEGUIBase-0_Static.lib "
-			"${CEGUI_INSTALL_PATH}/lib/CEGUICommonDialogs-0_Static.lib "
-			"${CEGUI_INSTALL_PATH}/lib/CEGUICoreWindowRendererSet_Static.lib "
-			"${CEGUI_INSTALL_PATH}/lib/CEGUIExpatParser_Static.lib "
-			"${CEGUI_INSTALL_PATH}/lib/CEGUIOpenGLRenderer-0_Static.lib "
-			"${CEGUI_DEPS_INSTALL_PATH}/lib/Expat.lib "
-			"${CEGUI_DEPS_INSTALL_PATH}/lib/Freetype.lib "
+			${CEGUI_INSTALL_PATH}/lib/CEGUIOpenGLRenderer-0_Static.lib
+			${CEGUI_INSTALL_PATH}/lib/CEGUIBase-0_Static.lib
+			${CEGUI_DEPS_INSTALL_PATH}/lib/static/glew.lib
+			${CEGUI_INSTALL_PATH}/lib/CEGUIExpatParser_Static.lib
+			${CEGUI_INSTALL_PATH}/lib/CEGUISTBImageCodec_Static.lib
+			${CEGUI_DEPS_INSTALL_PATH}/lib/static/freetype.lib
+			${CEGUI_INSTALL_PATH}/lib/CEGUICoreWindowRendererSet_Static.lib
+			${CEGUI_INSTALL_PATH}/lib/CEGUICommonDialogs-0_Static.lib
+			${CEGUI_DEPS_INSTALL_PATH}/lib/static/libexpat.lib
+			${CEGUI_DEPS_INSTALL_PATH}/lib/static/pcre.lib
 			PARENT_SCOPE
 		)
 	else()
 		set(CEGUI_LIBS
-			"${CEGUI_INSTALL_PATH}/lib/libCEGUIOpenGLRenderer-0_Static.a "
-			"${CEGUI_INSTALL_PATH}/lib/libCEGUIBase-0_Static.a "
-			"${CEGUI_DEPS_INSTALL_PATH}/lib/static/libglew.a"
-			"${CEGUI_INSTALL_PATH}/lib/libCEGUIExpatParser_Static.a "
-			"${CEGUI_INSTALL_PATH}/lib/libCEGUISTBImageCodec_Static.a "
-			"${CEGUI_DEPS_INSTALL_PATH}/lib/static/libfreetype.a"
-			"${CEGUI_INSTALL_PATH}/lib/libCEGUICoreWindowRendererSet_Static.a "
-			"${CEGUI_INSTALL_PATH}/lib/libCEGUICommonDialogs-0_Static.a "
-			"${CEGUI_DEPS_INSTALL_PATH}/lib/static/libexpat.a"
-			"${CEGUI_DEPS_INSTALL_PATH}/lib/static/libpcre.a"
+			${CEGUI_INSTALL_PATH}/lib/libCEGUIOpenGLRenderer-0_Static.a
+			${CEGUI_INSTALL_PATH}/lib/libCEGUIBase-0_Static.a
+			${CEGUI_DEPS_INSTALL_PATH}/lib/static/libglew.a
+			${CEGUI_INSTALL_PATH}/lib/libCEGUIExpatParser_Static.a
+			${CEGUI_INSTALL_PATH}/lib/libCEGUISTBImageCodec_Static.a
+			${CEGUI_DEPS_INSTALL_PATH}/lib/static/libfreetype.a
+			${CEGUI_INSTALL_PATH}/lib/libCEGUICoreWindowRendererSet_Static.a
+			${CEGUI_INSTALL_PATH}/lib/libCEGUICommonDialogs-0_Static.a
+			${CEGUI_DEPS_INSTALL_PATH}/lib/static/libexpat.a
+			${CEGUI_DEPS_INSTALL_PATH}/lib/static/libpcre.a
 			PARENT_SCOPE
 		)
 	endif()
 
+	install(
+		DIRECTORY ${CEGUI_DEPS_INSTALL_PATH}/include
+		DESTINATION ./include/irr/ext/GUI
```

Comment:
is the ext called `irr::ext::GUI` or `irr::ext::CEGUI` ?

Reply:
irr::ext::GUI means GUI dependencies used by ext.

Reply:
https://github.com/manhnt9/IrrlichtBAW/tree/sRGB/ext

A. There's no GUI folder here. 
B. With different GUI system out there, we cannot just use `GUI` as a name

Reply:
The only folders that should be in `./include` should match namespace names 100%

Reply:
Mis-typed, there's no namespace like irr::ext::GUI, only a GUI folder.

Reply:
i.e. if there's a `./include/irr/ext/VisibilityBuffer` then there is a `irr::ext::VisiblityBuffer`

Reply:
understood, see my other comment to solve the actual problem about this.